app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.kt

```diff
@@ -0,0 +1,195 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.utils.files
+
+import android.content.Context
+import androidx.test.InstrumentationRegistry
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.kiwix.kiwixmobile.core.utils.files.FileUtils.getAllZimParts
+import org.kiwix.kiwixmobile.core.utils.files.FileUtils.hasPart
+import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity
+import java.io.File
+import java.io.IOException
+import java.util.Random
+
+class FileUtilsInstrumentationTest {
+  private var context: Context? = null
+  private var testDir: File? = null
+  @Before fun executeBefore() {
+    context = InstrumentationRegistry.getTargetContext()
```

Comment:
This method is deprecated. Can we use this `ApplicationProvider.getApplicationContext()` instead?

Reply:
hi @gouri-panda ,

we can use ```androidx.test.platform.app.InstrumentationRegistry.getInstrumentation().targetContext``` insted of ```androidx.test.InstrumentationRegistry.getTargetContext()```

```ApplicationProvider.getApplicationContext()```  returns application context ```androidx.test.platform.app.InstrumentationRegistry.getInstrumentation().getTargetContext().getApplicationContext()```

Reply:
@MohitMaliDeveloper Sure use this package's API `androidx.test.platform.app`.

Reply:
hi @gouri-panda  done